x/staking/genesis.go

```diff
@@ -172,6 +177,7 @@ func ExportGenesis(ctx sdk.Context, keeper Keeper) types.GenesisState {
 		Delegations:          delegations,
 		UnbondingDelegations: unbondingDelegations,
 		Redelegations:        redelegations,
+		HistoricalInfos:      historicalInfos,
```

Comment:
nit (avoiding unnecessary var declarations)

```suggestion
		HistoricalInfos:      keeper.GetAllHistoricalInfo(ctx),
```